core/app/assets/stylesheets/refinery/sections/_menu_item_icons.scss

```diff
@@ -0,0 +1,9 @@
+.site-name {@include icon('bars', $plugin_icon_colour, $margin: 1.2em)}
+
+.refinery_images_icon {@include icon('photo', $plugin_icon_colour, $margin: 1.2em)}
```

Comment:
At this time, i don't like line 3 to 6, icons should be defined in plugin config only : https://github.com/refinery/refinerycms/pull/3201/files#diff-24e9d953dbbc6362d29db46264449211L15